Modules/System/Upgrade Tags/src/UpgradeTagImpl.Codeunit.al

```diff
@@ -29,6 +29,31 @@ codeunit 9996 "Upgrade Tag Impl."
         SetUpgradeTagForCompany(NewTag, CopyStr(CompanyName(), 1, MaxStrLen(ConstUpgradeTags.Company)));
     end;
 
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; SkipUpgrade: Boolean): Boolean
+    begin
+        exit(SetSkippedUpgrade(ExistingTag, CompanyName(), SkipUpgrade));
+    end;
+
+    procedure SetSkippedUpgrade(ExistingTag: Code[250]; TagCompanyName: Code[30]; SkipUpgrade: Boolean): Boolean
+    var
+        UpgradeTags: Record "Upgrade Tags";
+    begin
+        if UpgradeTags.Get(ExistingTag, TagCompanyName) then begin
+            UpgradeTags."Skipped Upgrade" := SkipUpgrade;
+            UpgradeTags.Modify();
+            exit(true);
+        end else
+            exit(false);
+    end;
+
+    procedure HasUpgradeTagSkipped(ExistingTag: Code[250]; TagCompanyName: Code[30]): Boolean
+    var
+        UpgradeTags: Record "Upgrade Tags";
+    begin
+        if UpgradeTags.Get(ExistingTag, TagCompanyName) then
+            exit(UpgradeTags."Skipped Upgrade");
+    end;
```

Comment:
Minor - add exit(false) so it is clear that the function should return false and it was not forgotten. It is good to have a clear intent

Reply:
Added exit false